server/routes/controllers/forgotPasswordController.js

```diff
@@ -20,23 +23,56 @@ const forgotPasswordController = {
     return Math.floor(Math.random() * 900000) + 100000 // generate a 6 digit code
   },
 
+  _getUser (email) {
+    return this.Users.findOne({email: email}).exec((err, forgot) => {
+      if (err) {
```

Comment:
this if/else statement can just be:

`return !!(err)`

or, if you prefer, the more verbose:

`return Boolean(err)`

Reply:
even just `!!err` works fine! this coerces to a boolean and negates it, then negates it back

Reply:
nice....to be honest I have some "issues" with that operator !! :P you probably won't see me using it a lot.